### PR TITLE
add labels to docker image

### PR DESCRIPTION
**Pull request by bgoncalv**

### CODE REVIEW

**Review by sgallagher — APPROVED** [review]

File: config.yaml [hunk 1]
```diff
@@ -13,8 +13,14 @@ module:
     docker:
         start:
         labels:
-            description: "I dont know"
-            io.k8s.description: "I dont know too"
+            com.redhat.component: "https://github.com/fedora-modularity/base-runtime"
```
Comment (TomasTomecek) [inline config.yaml]: please put a comment here saying this is a short-term solution since there is no bugzilla component yet

File: config.yaml [hunk 2]
```diff
@@ -13,8 +13,14 @@ module:
...
+            name: "base-runtime"
+            version: "master"
```
Comment (TomasTomecek) [inline config.yaml]: atm fedora guidelines are this should be set to 0

**Review by TomasTomecek — CHANGES_REQUESTED** [review]
`FGC` is still missing, see [container guidelines](https://fedoraproject.org/wiki/Container:Guidelines#LABELS). I would set it to `f26-boltron`.

**Comment by contyk** [comment]
+1

Thanks for the review, Tomas.

**Pull request merged**